Applications/spire/source/charting/charting_window.cpp

```diff
@@ -140,12 +144,22 @@ ChartingWindow::ChartingWindow(Ref<SecurityInputModel> input_model,
   m_security_widget->setFocus();
 }
 
-void ChartingWindow::set_model(std::shared_ptr<ChartModel> model) {
-  m_model = model;
+void ChartingWindow::set_models(std::shared_ptr<ChartModel> chart_model,
+    std::shared_ptr<TechnicalsModel> technicals_model) {
+  m_model = chart_model;
```

Comment:
```std::move(...);```

Reply:
Are both models moved?

Reply:
Think about it and propose a solution.

Reply:
I thought originally you said models can be shared?

Reply:
They are shared, yes.

Reply:
I think I've got it. They both should be moved.

Reply:
Okay, implement it that way and resubmit the PR.